app/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="nl">
   <head>
     <meta charset="utf-8">
-    <title>FrontendOrganizationPortal</title>
+    <title>OrganisatiePortaal</title>
```

Comment:
This is temporarily visible until the Ember code takes over, so better to make it consistent as well.